### ISSUE TITLE
Any plans on adding cpfr and cpto? 

### ISSUE THREAD

**Issue by fennectech** [jjolano/openps3ftp#9]
Being able to copy a game from a usb key while playing a game would be really nice.

**Comment by jjolano** [comment]
Sure, I'll see what I can do for version 4.4.

**Comment by oneohthree** [comment]
Won't the PS3 suffer (lagging for instance) when you copy and play a game at the same time?

**Comment by jjolano** [comment]
Yeah. It would mostly affect the hard drive load times. PS3 seems to allocate most of the processor to the game from what I've noticed during testing, but loading times definitely take a hit to performance.

**Comment by oneohthree** [comment]
So, IMHO this wouldn't benefit either. Right?

**Comment by jjolano** [comment]
It would pretty much still use the same resources as a normal transfer.
